Packs/rasterize/Integrations/rasterize/rasterize_test.py

```diff
@@ -364,3 +375,183 @@ def test_get_output_filenames():
+    mocker.patch.object(rasterize, 'get_chrome_instances_contents_dictionaries',
+                        return_value=[instance_id_to_chromes_options, chrome_options_to_port, instance_id_to_port])
+    mocker.patch.object(rasterize, 'is_chrome_running_locally', return_value=None)
+    mocker.patch.object(rasterize, 'terminate_chrome', return_value=None)
+    mocker.patch.object(rasterize, 'delete_row_with_old_chrome_configurations_from_chrome_instances_file', return_value=None)
+    mocker.patch.object(rasterize, 'generate_new_chrome_instance', return_value=["browser_object", "chrome_port"])
+    browser, chrome_port = chrome_manager()
+
+    assert browser == "browser_object"
+    assert chrome_port == "chrome_port"
+
+
+def test_chrome_options_in_chrome_options_and_instance_id_linked(mocker):
+    from rasterize import chrome_manager
+
+    instance_id = "22222222-2222-2222-2222-222222222222"  # exist
+    chrome_options = "chrome_options2"
+
+    mock_context = {
+        'context': {
+            'IntegrationInstanceID': instance_id
+        }
+    }
+
+    params = {
+        'chrome_options': chrome_options
+    }
+
+    mock_file_content = util_read_tsv("test_data/info.tsv")
+    mock_file_content_edited = mock_file_content.replace('\\t', '\t')
+    instance_id_to_chromes_options, chrome_options_to_port, instance_id_to_port = get_chrome_instances_contents_dictionaries(
+        mock_file_content_edited)
+    mocker.patch.object(demisto, 'callingContext', mock_context)
+    mocker.patch.object(demisto, 'params', return_value=params)
+    mocker.patch.object(rasterize, 'read_file', return_value=mock_file_content_edited)
+    mocker.patch.object(rasterize, 'get_chrome_instances_contents_dictionaries',
+                        return_value=[instance_id_to_chromes_options, chrome_options_to_port, instance_id_to_port])
+    mocker.patch.object(rasterize, 'is_chrome_running_locally', return_value="browser_object")
+    browser, chrome_port = chrome_manager()
+
+    assert browser == "browser_object"
+    assert chrome_port == "2222"
+
+
+def test_delete_row_with_old_chrome_configurations_from_info_file():
+    from rasterize import delete_row_with_old_chrome_configurations_from_chrome_instances_file
+
+    rasterize.CHROME_INSTANCES_FILE_PATH = "test_data/info.tsv"
+
+    mock_info = """2222\t22222222-2222-2222-2222-222222222222\tchrome_options2
+3333\t33333333-3333-3333-3333-333333333333\tchrome_options3
+test\ttesttest-test-test-test-testtesttest\tchrome_options0
+4444\t44444444-4444-4444-4444-444444444444\tchrome_options4
+"""
+    util_generate_mock_info_file(mock_info)
+    chrome_port_to_delete = "test"
+    instance_id_to_delete = "testtest-test-test-test-testtesttest"
+
+    mock_file_content = util_read_tsv("test_data/info.tsv")
+    mock_file_content_edited = mock_file_content.replace('\\t', '\t')
+
+    delete_row_with_old_chrome_configurations_from_chrome_instances_file(
```

Comment:
What are we desting here? where are the assertions? are we testing no error occurred?